Cargo.toml

```diff
@@ -34,14 +35,29 @@ all = "warn"
 [workspace.dependencies]
 kakarot-node = { path = "crates/node" }
 kakarot-pool = { path = "crates/pool" }
+kakarot-exex = { path = "crates/exex" }
 
 # Reth
-reth-transaction-pool = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0", default-features = false }
-reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0", default-features = false }
-reth-chainspec = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0", default-features = false }
-reth-node-ethereum = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0", default-features = false }
-reth-node-builder = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0", default-features = false }
-reth-node-api = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0", default-features = false }
-reth-ethereum-engine-primitives = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0", default-features = false }
+reth-transaction-pool = { git = "https://github.com/paradigmxyz/reth.git" }
```

Comment:
@greged93 Here I just put the github url without any particular flag because the versions evolve very quickly and for this wip project in my opinion it is more practical to always be up to date without having to manage the flags everywhere every week.

See https://github.com/paradigmxyz/reth-exex-examples/blob/0c7cf9f9e7195a07366dda1f33f51a6f9b60d972/Cargo.toml#L22

Reply:
ok!